eZ/Bundle/EzPublishCoreBundle/Resources/config/fieldtype_services.yml

```diff
@@ -12,6 +12,256 @@ parameters:
     ezpublish.fieldType.ezpage.hashConverter.class: eZ\Publish\Core\FieldType\Page\HashConverter
 
     # RichText
+    ezpublish.fieldType.ezrichtext.normalizer.character_entity_definition.class: eZ\Publish\Core\FieldType\RichText\Normalizer\CharacterEntityDefinition
+    ezpublish.fieldType.ezrichtext.normalizer.html_entities.data:
+        nbsp: 160
```

Comment:
Added DTD file, changed normalizer from `CharacterEntityDefinition` to `DocumentTypeDefinition` and rebased.